### ISSUE TITLE
No support for multiple parents with multiple children

### ISSUE THREAD

**Issue by danbeam** [danbeam/oo-css#1]
Currently there is support for the following use case:

```
parent {
    /* style here */
    child1, child2 {
        /* style here */
    }
}
```

to be transformed to this:

```
parent child 1, parent child 2 {
    /* style here */
}
```

I'd like this to work with multiple parents to multiple children

```
parent1, parent2 {
    /* style here */
    child {
        /* style here */
    }
}
```

to go to this ->

```
parent1 child, parent2 child {
    /* style here */
}
```

as well as

```
parent1, parent2 {
    /* style here */
    child1, child2 {
        /* style here */
    }
}
```

to be re-arranged to

```
parent1 child1, parent1 child2, parent2 child1, parent2 child2 {
    /* style here */
}
```

and so on.

**Comment by danbeam** [comment]
Fixed in d6491e46da3c8d11b308024ebe861af2ed60bc82.

**Comment by danbeam** [comment]
Marking fixed for now.
